src/rpc/blockchain.cpp

```diff
@@ -794,7 +794,7 @@ static RPCHelpMan getblockfrompeer()
     ChainstateManager& chainman = EnsureChainman(node);
     PeerManager& peerman = EnsurePeerman(node);
 
-    const uint256& block_hash{ParseHashV(request.params[0], "block_hash")};
+    const uint256& block_hash{ParseHashV(request.params[0], "blockhash")};
```

Comment:
is it breaking change? should we have release notes?

Reply:
I don't think this is breaking; it's only the name of the param, nothing more. No way this could break someone's flow

Reply:
well, if someone is using named params it would be indeed a breaking change imo

Reply:
can you do named params with rpc?

Reply:
```
> dash-cli -named getblockfrompeer block_hash=000000000000002ef98b2f8679bf97c84e201de628e9cd59985c559e426c477d peer_id=1
{
  "warnings": "Block already downloaded"
}
```

Reply:
```
> dash-cli -named getblockfrompeer blockhash=000000000000002ef98b2f8679bf97c84e201de628e9cd59985c559e426c477d peer_id=1
error code: -8
error message:
Unknown named parameter blockhash
```
(on develop)

Reply:
yes, of course, all of our functional tests do that. For example: https://github.com/dashpay/dash/blob/master/test/functional/feature_asset_locks.py#L207

Reply:
release notes pushed :)